cms/static/js/views/settings/main.js

```diff
@@ -164,7 +209,26 @@ var DetailsView = ValidatingView.extend({
         $(e.currentTarget).attr('title', currentTimeText);
     },
     updateModel: function(event) {
+        var value;
+        var index = event.currentTarget.getAttribute('data-index');
         switch (event.currentTarget.id) {
+        case 'course-learning-info-' + index:
```

Comment:
But learning_info is also updated in learning_info.js (when learning infos are removed), and instructor_info is also updated in instructor_info.js (changing instructors, removing instructors). Do those updates get double-updated here? What things have to be handled here?